src/models/Capability/Repository.ts

```diff
@@ -12,15 +12,23 @@ export const CapabilityRepository = {
     Capability.findAll({ where: { description: { [Op.or]: description }} }),
   findOrCreate: async (description: string) =>
     Capability.findOrCreate({ where: { description } }),
+  findOrCreateByDescriptions: async (descriptions: string[] = []) => {
+    const capabilities: Capability[] = [];
+    for (const description of descriptions) {
+      const result = await CapabilityRepository.findOrCreate(description);
+      capabilities.push(result[0]);
+    }
+    return capabilities;
+  },
   findAll: async () =>
     Capability.findAll(),
   deleteByCode: async (code: string) => {
     const transaction = await Database.transaction();
     try {
       await ApplicantCapability.destroy({ where: { capabilityUuid: code }, transaction});
-      const carrerDestroyed = await Capability.destroy({ where: { code }, transaction });
+      const careerDestroyed = await Capability.destroy({ where: { code }, transaction });
       await transaction.commit();
-      return carrerDestroyed;
+      return careerDestroyed;
```

Comment:
no retornaria el record destruido, no tiene mucha utilidad eso (de todos modos, sería destroyedCareer)

Reply:
Si, igual es raro, solo hice in fix de un typo, pero ese metodo litteral no se usa ni se testea jaja

Reply:
Lo borraria pero no se por que esta

Reply:
👍